style.css

```diff
@@ -601,11 +600,13 @@ input:focus:invalid {
  #container1 {
     height: 100vh;
     background-color: #1c1a19;
-    background-image: url("images/Illustration-header.png");
+    background-image: url(images/Illustration-header.png);
     background-repeat: no-repeat;
     background-position: right;
+    background-position-y: 0.2rem;
     background-size: 40%;
     padding-bottom: 3rem;
```

Comment:
- [ ] As requested by the previous reviewer you should stick to the design as much as possible. like color, spacing, font, etc. Please adjust he spacing and stick to the design as much as possible.
### Your design
![his](https://github.com/Tshepo45/Portfolio-desktop-version/assets/81727988/1ab1979f-18f2-4c48-a214-62262b54eb63)

### Figma design
![fimga](https://github.com/Tshepo45/Portfolio-desktop-version/assets/81727988/067d9560-4386-4f93-93dc-76ffb1d00dbe)